src/templates/blog.tsx

```diff
@@ -23,11 +23,9 @@ export default function blog({ location, data }: PageProps<BlogQuery, {}>) {
         >
             <div className="md:px-4 mt-12 py-6 md:w-11/12 mx-auto">
                 <div className="mx-auto relative">
-                    <Img
-                        fluid={
-                            data.mdx.frontmatter.banner.childImageSharp.fluid
-                        }
-                    />
+                    {data.mdx.frontmatter.banner.publicURL.endsWith('.svg') ?
+                    <img src={data.mdx.frontmatter.banner.publicURL} alt="''"/> :
+                    <Img fluid={data.mdx.frontmatter.banner.childImageSharp.fluid}/>}

```

Comment:
We can build this as a component to proxy this logic in the general sense.

Reply:
I think that would be awesome!